src/v2/Apps/Consign/Components/InDemandNow/ConsignTopArtists.tsx

```diff
@@ -1,81 +1,145 @@
 import React from "react"
-import { Box, Flex, Join, Separator, Spacer, Text } from "@artsy/palette"
 import { SystemQueryRenderer as QueryRenderer } from "v2/Artsy/Relay/SystemQueryRenderer"
 import { useSystemContext } from "v2/Artsy"
 import { graphql } from "react-relay"
-import { shuffle } from "lodash"
-
+import { chunk, shuffle } from "lodash"
 import { ConsignTopArtistsQuery } from "v2/__generated__/ConsignTopArtistsQuery.graphql"
+import styled from "styled-components"
 
-export const ConsignTopArtists: React.FC = () => {
-  const { relayEnvironment } = useSystemContext()
+import {
+  Avatar,
+  Text as BaseText,
+  Box,
+  Flex,
+  Join,
+  Spacer,
+} from "@artsy/palette"
+
+type ArtworkProps = ConsignTopArtistsQuery["response"]["targetSupply"]["microfunnel"][0]["artworksConnection"]["edges"][0]["node"]
 
+export const ConsignTopArtists: React.FC = () => {
   return (
-    <Box>
+    <>
       <Box>
-        <Text variant="title">Top artists</Text>
-        <Separator />
+        <Text textAlign={"left"} mb={2} variant="largeTitle">
+          Top Artists
+        </Text>
       </Box>
 
-      <Box>
-        <QueryRenderer<ConsignTopArtistsQuery>
-          environment={relayEnvironment}
-          variables={{}}
-          query={graphql`
-            query ConsignTopArtistsQuery {
-              targetSupply {
-                microfunnel {
-                  artworksConnection(first: 1) {
-                    edges {
-                      node {
-                        slug
-                        internalID
-                        href
-                        artistNames
-                        image {
-                          imageURL
-                        }
-                        realizedPrice
-                      }
+      <ConsignTopArtistsQueryRenderer />
+    </>
+  )
+}
+
+const ConsignTopArtistsQueryRenderer: React.FC = () => {
+  const { relayEnvironment } = useSystemContext()
+
+  return (
+    <QueryRenderer<ConsignTopArtistsQuery>
```

Comment:
(non-blocking comment) I'm using effectively the same query to get the list of recently sold artworks for the artwork rail so just a heads up that one of my next steps will be to reuse this for that.

Reply:
Sounds good, yeah, one of the strange things about relay is that you can share queries from query renderer's, but you can't share queries from fragement containers.